src/components/AdminPage/AdminPanel/AdminPanel.js

```diff
@@ -1,5 +1,114 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { useCookies } from "react-cookie";
+import styles from "./AdminPanel.module.css";
+import { auth } from "../../../firebase";
+import { signInWithEmailAndPassword } from "firebase/auth";
+import { Button, Form, FormGroup } from "react-bootstrap";
```

Comment:
Remove unused imports and unused vars.

Reply:
I don't see any useless imports or vars in this file. What do you mean?